src/modelinstance.cpp

```diff
@@ -1459,43 +1459,58 @@ Status ModelInstance::inferAsync(const RequestType* requestProto,
     // set callback
     // TODO check if there is callback in async
     OVMS_InferenceRequestCompletionCallback_t userCallback = requestProto->getResponseCompleteCallback();
+    if (userCallback == nullptr) {
```

Comment:
@atobiszei should we enforce the callback? does OV enforce setting callback? It might be useful to not set it for warmups?